gst/edge/edge_elements.c

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2022 Samsung Electronics Co., Ltd.
+ *
+ * @file    edge_sink.h
+ * @date    02 Aug 2022
+ * @brief   Register edge plugins
+ * @author  Yechan Choi <yechan9.choi@samsung.com>
+ * @see     http://github.com/nnstreamer/nnstreamer
+ * @bug     No known bugs
+ *
+ */
+#include <gst/gst.h>
+
+#include "edge_common.h"
+#include "edge_sink.h"
+#include "edge_src.h"
```

Comment:
Is ```edge_common.h``` included in ```edge_sink.h``` and ```edge_src.h``` ? Then, you don't need to include ```edge_common.h``` here.

Reply:
Removed. Thanks.